static/src/stylesheets/base/_inline-svgs.scss

```diff
@@ -6,6 +6,15 @@
     }
 }
 
+.inline-icon--light-grey {
+    fill: colour(neutral-3);
+}
+
+.inline-close.small {
```

Comment:
sorry I missed that whole PR going out, I think the same applies to the .small as the .light-grey about BEM syntax